src/pypistats/__init__.py

```diff
@@ -124,6 +124,9 @@ def pypi_stats_api(
 
         _save_cache(cache_file, res)
 
+    if not res.get("data", []):
+        return f"The package '{res.get('package', '')}' does not exist"
```

Comment:
This also happens when looking up a new package that hasn't had any downloads yet. I think we won't see any numbers until the next day. 

For example, looking at the new packages feed: https://pypi.org/rss/packages.xml

https://pypi.org/project/hnt-sap-service-library/ is a new package with no downloads registered yet and also triggers this. 

Shall we update the message?

```suggestion
        return f"No data found for '{res.get('package', '')}'"
```

Or maybe link to it:

```suggestion
        return f"No data found for https://pypi.org/project/{res.get('package', '')}/"
```

Reply:
You are right. I noticed this issue. 
I think there should be some downloads on the first day from mirrors. 
I reckon this tool is still very useful. 
I prefer the second one. Great suggestion!